src/Glpi/Kernel/Kernel.php

```diff
@@ -52,6 +52,15 @@ final class Kernel extends BaseKernel
 
     public function __construct(?string $env = null)
     {
+        if ($env !== null) {
+            define('GLPI_ENVIRONMENT_TYPE', $env);
+        }
+
+        // Initialize configuration constants.
+        // It must be done after the autoload inclusion that requires some constants to be defined (e.g. GLPI_VERSION).
+        // It must be done before the Kernel boot as some of the define constants must be defined during the boot sequence.
+        (new ConfigurationConstants(dirname(__DIR__, 3)))->computeConstants();
```

Comment:
Why not use project dir?
```suggestion
        (new ConfigurationConstants($this->getProjectDir()))->computeConstants();
```

Reply:
done